code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -196,6 +195,15 @@
 			to_chat(src, "<span class='warning'>Something in this place is interfering with our link to Queen Mother. We are unable to evolve to a psychic caste here!</span>")
 			return
 
+	else if(new_caste_type == /mob/living/carbon/xenomorph/hivemind) //Special case for dealing with hiveminds - this may be subject to heavy change, such as multiple hiveminds potentially being an option
```

Comment:
```suggestion
	else if(new_caste_type == /mob/living/carbon/xenomorph/hivemind) // TODO: Refactor this to support multiple hiveminds``

Reply:
this isnt a TODO yet, multiple hiveminds is an idea im still bouncing about and will require a heavy refactor of hivemind abilities

Reply:
Either commit it or dont, the comment isn't helping in its current state.